cmd/filegen/main.go

```diff
@@ -85,6 +85,15 @@ VERSION:
 		Usage: "Number of initial metachain observers, private/public keys, to generate",
 		Value: 1,
 	}
+	hysteresis = cli.Float64Flag{
+		Name:  "hysteresis",
+		Usage: "Hysteresis value",
+		Value: 0.0,
+	}
+	adaptivity = cli.BoolFlag{
+		Name:  "adaptivity",
+		Usage: "adaptivity value",
```

Comment:
true if shard merging and splitting is enabled

Reply:
done